README.md

```diff
@@ -77,8 +77,14 @@ resources subdirectory: `./site_root/input/resources`.
 
 3. Validate the models
 
-   `fhirmodel validate ./site_root/ig.ini --publisher_opts='-tx n/a'`
+   ```shell
+   # Method 1 - Uses the dockerized IG publisher
+   fhirmodel validate ./site_root/ig.ini --publisher_opts='-tx n/a'
 
+   # Method 2 - Uses native IG publisher - faster than above method
+   fhirmodel add ./site_root/input/resources
```

Comment:
Previously, it didn't recursively look into all sub-directories/-files. Does it now? If not, we may want to describe that.